buildtest/cli/compilers.py

```diff
@@ -34,6 +40,42 @@ def compiler_cmd(args, configuration):
         bc.print_yaml()
 
 
+def compiler_test(args, configuration):
+    """This method implements ``buildtest config compilers test`` which tests
+    the compilers with the corresponding modules if set. This command iterates
+    over all compilers and perform the module load test and show an output of
+    each compiler with a PASS or FAIL next to each compiler section.
```

Comment:
you can remove `with a PASS or FAIL next to each compiler section.` since we are showing a check mark or a `x`